app/src/main/java/com/example/hangouts/BayesianRating.java

```diff
@@ -24,14 +25,91 @@ public static double getBayesianRating(List<Double> list, double significance) {
         if (sum == 0) {
             return 0;
         }
-        double z_value = getPpf(1 - (1 - significance) / 2);
+        double zValue = getPpf(1 - (1 - significance) / 2);
         double firstTerm = 0;
         double secondTerm = 0;
         for (int i = 0; i < size; i++) {
             firstTerm += (i + 1) * (list.get(i) + 1) / (sum + size);
             secondTerm += (i + 1) * (i + 1) * (list.get(i) + 1) / (sum + size);
         }
-        return firstTerm - z_value *
-                Math.sqrt((secondTerm - firstTerm*firstTerm)/(sum + size + 1));
+        return firstTerm - zValue *
+                Math.sqrt((secondTerm - firstTerm * firstTerm) / (sum + size + 1));
+    }
+
+    private static double inverseCumulativeProbability(double x) {
+        return FastMath.sqrt(2.0) * inverseErrorFunction(2 * x - 1);
+    }
+
+    /***
+     * The algorithm to approximate the error inverse function with the finite series method
+     * would look like this, however not only this is not computationally inefficient,
+     * the precision of the double data type won't be able to handle the calculations after the
+     * third term.
+     * final static double SQRTPI_OVER_TWO = FastMath.sqrt(FastMath.PI) / 2;
+     *  final double epsilon = 0.001;
+     *         double diff = 1;
+     *
+     *         ArrayList<Double> terms = new ArrayList<Double>();
+     *         terms.add(SQRTPI_OVER_TWO * x);
+     *         double sum = 0.0;
+     *         // First Term
+     *         double prevTerm = SQRTPI_OVER_TWO;
+     *         // c is numerator
+     *         List<Double> c = new ArrayList<>();
+     *         c.add(1.0);
+     *         int k = 1;
+     *         while (diff > epsilon) {
+     *
+     *             // get numerator
+     *             System.out.println("k = " + k);
+     *             double cSum = 0;
+     *             for (int m = 0; m < k; m++) {
+     *                 System.out.println("m = " + m);
+     *                 double newTerm = (c.get(m) * c.get(k - 1 - m)) / ((m + 1) * (2 * m + 1));
+     *                 cSum += newTerm;
+     *                 c.add(newTerm);
+     *             }
+     *
+     *             double newTerm = (cSum / (2 * k + 1)) * FastMath.pow((SQRTPI_OVER_TWO) * x,
+     *                     (2 * k + 1));
+     *             sum += newTerm;
+     *             diff = prevTerm - newTerm;
+     *             prevTerm = newTerm;
+     *             System.out.println("New Term = " + newTerm);
+     *             k++;
+     *         }
+     *         return sum;
+     */
+
+    /***
+     * Approximating the inverse error function with the Taylor Finite Series method,
+     * first 23 terms.
+     */
+    private static double inverseErrorFunction(double x) {
+        return 0.8862269254527579 * x
```

Comment:
Some comment as to where the numbers came from (hand calculated on specialized calc) would be helpful.  
Also looks like a repetitive pattern  - n-array of constants applying same function based on the index.  There are some ways to 'reduce' this.  In kotlin and functional programming there is literally a 'reduce' function that you could use.  Perhaps a java 8 equivalent - or just a loop.  
In other words - would putting all the constants in an static final array and then looping make this more concise/less errorprone?